### PR TITLE
bug 1797399 correcting GCP install-config.yaml

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1797399

@gpei, will you PTAL?

### CODE REVIEW

File: modules/installation-gcp-config-yaml.adoc [hunk 1]
```diff
@@ -37,21 +37,15 @@ controlPlane: <2>
   name: master
   platform:
     gcp:
-      rootVolume:
-        iops: 4000
-        size: 500
-        type: io1
+      project: project-name
```
Comment (shellyyang1989) [inline modules/installation-gcp-config-yaml.adoc]: Per https://github.com/openshift/installer/blob/master/docs/user/gcp/customization.md#machine-pools, machine pools for GCP support **type** and **zones** only without **project**. The available zones for a specific region can be found https://cloud.google.com/compute/docs/regions-zones

File: modules/installation-gcp-config-yaml.adoc [hunk 2]
```diff
@@ -37,21 +37,15 @@ controlPlane: <2>
...
 compute: <2>
 - hyperthreading: Enabled <3>
   name: worker
   platform:
     gcp:
-      rootVolume:
-        iops: 2000
-        size: 500
-        type: io1 <5>
+      project: project-name
```
Comment (shellyyang1989) [inline modules/installation-gcp-config-yaml.adoc]: Same as above

**Comment by kalexand-rh** [comment]
@shellyyang1989, I've updated the file. Will you please take another look?

File: modules/installation-gcp-config-yaml.adoc [hunk 3]
```diff
@@ -37,21 +37,13 @@ controlPlane: <2>
   name: master
   platform:
     gcp:
-      rootVolume:
-        iops: 4000
-        size: 500
-        type: io1
       type: n1-standard-4
```
Comment (shellyyang1989) [inline modules/installation-gcp-config-yaml.adoc]: GCP allows user to modify zones hence suggesting to add zones, e.g.
platform:
  gcp:
    type: n1-standard-4
    zones: us-central1-a

**Comment by shellyyang1989** [comment]
Another thing is that us-central-1 is not an available GCP region, please update it to us-central1. Thanks
region: us-central-1 (1)

**Comment by kalexand-rh** [comment]
@shellyyang1989, will you please take another look?

File: modules/installation-configuration-parameters.adoc [hunk 4]
```diff
@@ -384,6 +384,16 @@ ifdef::gcp[]
 |The name of the existing VPC that you want to deploy your cluster to.
 |String.
 
+|`platform.gcp.type`
+|The link:https://cloud.google.com/compute/docs/machine-types[GCP machine type].
+|The subnet name.
```
Comment (shellyyang1989) [inline modules/installation-configuration-parameters.adoc]: > |The subnet name.
Does it mean the description for platform.gcp.type? If so, please use The GCP machine type instead

File: modules/installation-configuration-parameters.adoc [hunk 5]
```diff
@@ -384,6 +384,16 @@ ifdef::gcp[]
...
+
+|`platform.gcp.zones`
+|The availability zones where the installation program creates machines for the
+control plane MachinePool.
```
Comment (shellyyang1989) [inline modules/installation-configuration-parameters.adoc]: > control plane MachinePool
The zone is not only used for control plane machinepool, but also compute nodes machinepool

File: modules/installation-gcp-config-yaml.adoc [hunk 6]
```diff
@@ -72,23 +70,23 @@ endif::[]
 platform:
   gcp:
     ProjectID: openshift-production <1>
-    region: us-central-1 <1>
+    region: us-central-1a <1>
```
Comment (shellyyang1989) [inline modules/installation-gcp-config-yaml.adoc]: us-central-1a is not a valid region for gcp, please use us-central1 instead. Please check out https://cloud.google.com/compute/docs/regions-zones#available for all available regions

**Comment by kalexand-rh** [comment]
@shellyyang1989, will you please take another look?

File: modules/installation-gcp-config-yaml.adoc [hunk 7]
```diff
@@ -72,23 +70,23 @@ endif::[]
 platform:
   gcp:
     ProjectID: openshift-production <1>
-    region: us-central-1 <1>
+    region: us-central1-a <1>
```
Comment (shellyyang1989) [inline modules/installation-gcp-config-yaml.adoc]: us-central1-a is a zone rather than a region. Please update it to us-central1. Thanks

Comment (kalexand-rh) [reply modules/installation-gcp-config-yaml.adoc]: @shellyyang1989, thank you!

**Comment by shellyyang1989** [comment]
@kalexand-rh LGTM. Thanks

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL?

File: modules/installation-gcp-config-yaml.adoc [hunk 8]
```diff
@@ -100,25 +98,24 @@ endif::private[]
...
+<8> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
+<9> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 endif::vpc[]
 ifndef::vpc[]
-<6> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
-<7> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
+<5> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
```
Comment (bobfuru) [inline modules/installation-gcp-config-yaml.adoc]: Is there any way you can break down the third sentence? With all the proper names, it's a little hard to tell who is doing what to whom. :)

File: modules/installation-gcp-config-yaml.adoc [hunk 9]
```diff
@@ -100,25 +98,24 @@ endif::private[]
...
+<6> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 endif::vpc[]
 +
 [NOTE]
 ====
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery on, specify an SSH key that your `ssh-agent` process uses.
 ====
 ifdef::private[]
-<11> How to publish the user-facing endpoints of your cluster. Set `publish` to `Internal` to deploy a private cluster, which cannot be accessed from the internet. The default value is `External`.
+<10> How to publish the user-facing endpoints of your cluster. Set `publish` to `Internal` to deploy a private cluster, which cannot be accessed from the internet. The default value is `External`.
```
Comment (bobfuru) [inline modules/installation-gcp-config-yaml.adoc]: s/cluster, which cannot/cluster that cannot

**Review by bobfuru — COMMENTED** [review]
Couple of tiny nits but otherwise LGTM!

**Pull request merged**